apps/web/src/pages/templates/workflow/workflow/node-types/WorkflowNode.tsx

```diff
@@ -1,14 +1,8 @@
+/* eslint-disable max-len */
```

Comment:
you don't need to disable this rule, just split big string into two and concatenate ;)